packages/app-portal/src/systems/Bridge/pages/Bridge.tsx

```diff
@@ -122,12 +121,13 @@ export const Bridge = () => {
           Any assets deposited to Fuel can take up to {timeToWithdrawFormatted}{' '}
           to withdraw back to Ethereum. Learn more about our architecture and
           security in our&nbsp;
-          <Link
-            isExternal
+          <a
```

Comment:
We just need change to keep using that `Link`.
It adds a pretty icon and other styles to it.

You can keep `target` and `rel` as you did. That looks good.

```suggestion
          <Link
```

Reply:
Done.